mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -517,7 +527,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_5809 isOnFire ()Z
 	METHOD method_5810 isPushable ()Z
 	METHOD method_5812 getPosVector ()Lnet/minecraft/class_243;
-	METHOD method_5814 setPosition (DDD)V
+	METHOD method_5814 initPosition (DDD)V
```

Comment:
I'm not sure if `init` is a good verb here. It's also used in ticking.

Reply:
Also what makes it "not a setter"?

Reply:
it is called in the entity spawn packet for client entity creation, so...